Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -0,0 +1,53 @@
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol CommentRepliesViewModelInputs {
+  /// Call with the comment/project that we are viewing replies for. Both can be provided to minimize
+  /// the number of API requests made, but it will be assumed we are viewing the replies for the comment.
+  func configureWith(comment: Comment, project: Project)
+
+  /// Call when the view loads.
+  func viewDidLoad()
+}
+
+public protocol CommentRepliesViewModelOutputs {
+  // TODO: Likely we will be adding `[Comment]` to this output to load replies
+  /// Emits a root `Comment`s and the `Project` to load into the data source.
```

Comment:
Maybe something like:

Emits a tuple of `Comment` and `Project` to load into the data source.

Reply:
Removed instances of `Project` because it isn't need for the main query of this VM.